package.json

```diff
@@ -43,8 +43,10 @@
     "react-dom": "16.13.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-40.0.1.tar.gz",
     "react-native-dialog": "^8.0.1",
+    "react-native-dropdown-picker": "^5.1.21",
```

Comment:
Do we still need this library? It appears to be imported into `CollectionInputScreen` but the `DropDownPicker` component was commented out.

Reply:
removed